Total_deaths fitness function

Pimski26 opened the pull request:
Implemented a total_deaths.m cost function under +lib/+costs

Tested this function under +tests/+lib/+costs/total_deaths_test.m

Pimski26 committed:
Total_deaths fitness function

Implemented a total_deaths.m cost function under +lib/+costs

Tested this function under +tests/+lib/+costs/total_deaths_test.m

roelhem commented on +tests/+lib/+costs/total_deaths_test.m:
```diff
@@ -0,0 +1,21 @@
+    'StartDate', '2020-10-09', ... The date for which the data should be loaded.
+    'Steps', 160 , ... The amount of steps to calculate beforehand.
+    'Method', "EulerForward", ... The numeric method to solve the ODE of the model.
+    'AgeGroups', 0:10:70, ... The way the age groups should be devided.
```
You just input the default values of `Method`, `AgeGroups` and `ContactMatrixFile` here. 

You can leave them out if you want, but no harm is done to explicitly set them.

roelhem commented on +lib/+costs/total_deaths.m:
```diff
@@ -0,0 +1,11 @@
+function [nr_of_deaths] = total_deaths(M,l)
```
The `[nr_of_deaths]` here tells matlab that this method **has to** have a result variable to run line 8. 

This is not the case for this function. It is therefore best practice to change `[nr_of_deaths]` to `nr_of_deaths`.

Again, for this use case, it does no harm if you leave it this way....

roelhem reviewed (commented):
I added some best-practice comments on the code.

The code will work just fine this way, but it might be a good idea to change those things.

Pimski26 committed:
Tidying code

Removed square brackets from function output

Removed default arguments when initiating ModelResult instance

Pimski26 committed:
Changed name of test script

Changed the name of +tests/+lib/+costs/total_deaths_test.m to +tests/+lib/+costs/test_total_deaths.m

roelhem reviewed (approved):
Nice!
